operator/pkg/securedcluster/values/translation/translation.go

```diff
@@ -113,6 +119,9 @@ func (t Translator) translate(ctx context.Context, sc platform.SecuredCluster) (
 	}
 
 	v.AddChild("scanner", t.getLocalScannerComponentValues(sc, scannerAutoSenseConfig))
+	if sc.Spec.ScannerV4 != nil && features.ScannerV4Support.Enabled() {
```

Comment:
General remark: I fear we need to discuss a little bit more about the exact semantics of scanner (v4) activation and defaulting. Background: it is not clear to me (yet) how the operator-way and the Helm chart (which also includes logic for figuring out which components to deploy) interact or overlap.

We can do this on a different ticket.

Reply:
Feel free to resolve.

Reply:
Matches my gut feeling, let's discuss this and open another PR if we'd like a change.